README.md

```diff
@@ -0,0 +1,162 @@
+<p align="center">
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="./.github/logo_dark.svg">
+    <source media="(prefers-color-scheme: light)" srcset="./.github/logo_light.svg">
+    <img src="./.github/logo_light.svg"  width="500px" alt="NeoFS logo">
+  </picture>
+</p>
+<p align="center">
+  <a href="https://fs.neo.org">NeoFS</a> is a decentralized distributed object storage integrated with the <a href="https://neo.org">NEO Blockchain</a>.
+</p>
+
+# GitHub Action to Publish Allure report from nspcc-dev projects to NeoFS
+This action allows [Allure reports](https://github.com/allure-framework/allure2) of tests running in Neo SPCC
```

Comment:
Just NeoSPCC repos?

Reply:
> Just NeoSPCC repos?

There is a lot specific to our repositories here.
Or do you want to remove the mention of NSPCC so that third party users are not afraid to try this action?

Reply:
To me there shouldn't be a lot of it. Suppose I'm @AxLabs and I want to use this for Grantshares tests. In theory it should be possible to do that. Is there anything not allowing to do that?

Reply:
> To me there shouldn't be a lot of it. Suppose I'm @AxLabs and I want to use this for Grantshares tests. In theory it should be possible to do that. Is there anything not allowing to do that?

Fixed